src/components/Home/index.test.tsx

```diff
@@ -66,7 +66,8 @@ it('shows port for emulator that are loaded', () => {
   expect(getByText(/9000/)).not.toBeNull();
 });
 
-it('renders error message when errored', () => {
+// Updated error screen is WIP for Home for now.
+it.skip('renders error message when errored', () => {
```

Comment:
remove .skip

Reply:
This is intentionally skipped (based on the comment above) and I'll redo the test in the next PR.

Reply:
Oh, I see :) maybe put `// intentional skip:` or something like that.